inst/examples/installBioc.r

```diff
@@ -24,20 +34,93 @@ package 'BiocManger' which has be installed.
 
 Examples:
   installBioc.r -l /tmp/lib S4Vectors               # install into given library
+  installBioc.r --update Biobase                    # install package and update older packages
+  installBioc.r --deps NA --error --skipinstalled   # install package without suggested dependencies, throw an error on installation failure and skip packages that are already present
```

Comment:
`docopt` prints this line on help so maybe doing 

```
   installBioc.r --deps NA --error --skipinstalled   # install package without suggested dependencies, throw an error
                                                                           # on installation failure and skip packages that are already present
```

(with possible whitespace / space adjustment) is nicer?

Reply:
I am adjusting this now.